tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaApisTest.java

```diff
@@ -466,7 +469,7 @@ public void testBrokerRespectsPartitionsOrderAndSizeLimits() throws Exception {
 
         int messagesPerPartition = 9;
         int maxResponseBytes = 800;
-        int maxPartitionBytes = 900;
+        int maxPartitionBytes = 1000;
```

Comment:
`testBrokerRespectsPartitionsOrderAndSizeLimits` is ignored now, changes here are meaningless. Or do you confirm that this change fixes the test?

Reply:
I changed the `PulsarEntryFormatter#decode` method, make one MemoryRecords contain multiple batches, so there will add more header size than one batch header size, the 900 was not enough, the max partition bytes limit process needs more work.

Reply:
If the test could pass, please remove the `@Ignore` and verify it. Otherwise, we should keep the test unchanged and fix it later.

Reply:
Ok, I'll revert the change for this test and retain the `@Ignore` annotation.